kalita-website/content/docs/atomic-swap/_atomic.md

```diff
@@ -0,0 +1,112 @@
++++
+title = "Atomic Swaps"
+description = "How atomic swap works and why it is cool"
+date = 2021-05-01T19:30:00+00:00
+updated = 2021-05-01T19:30:00+00:00
+draft = false
+weight = 30
+sort_by = "weight"
+template = "docs/page.html"
+
+[extra]
+lead = ""
+toc = true
+top = false
++++
+
+
+### Introdution
+An atomic swap is a smart contract technology that enables the exchange of one cryptocurrency for another without using centralized intermediaries, such as exchanges. This article will show the atomic swap between ergo and bitcoin.
```

Comment:
An atomic swap is a smart contract technology that provides a way to exchange one cryptocurrency for another without using centralized intermediaries, such as exchanges. Here we describe atomic swap protocol by example between Ergo and Bitcoin.